### DISCUSSION TITLE
Unable to retrieve package details from GraphQL query as of 8/2

### DISCUSSION THREAD

**Discussion by kspearsempire** [community/community#29589] [Actions] [answered]
We are not able to retrieve package details from GraphQL query

Here is existing curl statement to get specific version

**Request**
`curl -H "Authorization: bearer <<access_token>>" -d " 
	{ \"query\": \"query { repository(owner: \\\"ORG_NAME_HERE\\\", name: \\\"NAME_HERE\\\") 
	{ packages(names: \\\"NAME_HERE\\\", first: 1) 
	{ nodes {version(version: \\\"1.0.0\\\"){id,version } } } } }\" } " 
	https://api.github.com/graphql
`
**Response**
`{"data":{"repository":{"packages":{"nodes":[]}}}}`

This issue has been occurring since 2nd August 2022.

We are getting blank nodes in response, however we are seeing proper response from the same query for packages which were published before 2nd August 2022..

**Request**
`curl -H "Authorization: bearer <<access_token>>" -d " 
	{ \"query\": \"query { repository(owner: \\\"ORG_NAME_HERE\\\", name: \\\"NAME_HERE\\\") 
	{ packages(names: \\\"NAME_HERE\\\", first: 1) 
	{ nodes {version(version: \\\"1.0.0\\\"){id,version } } } } }\" } " 
	https://api.github.com/graphql
`

**Response**
`{"data":{"repository":{"packages":{"nodes":[{"version":{"id":"ID_WAS_RETURNED_HERE","version":"1.0.0"}}]}}}}`

We are able to publish new packages with new version on registry https://npm.pkg.github.com/ORG_NAME_HERE

When we access https://npm.pkg.github.com/ORG_NAME_HERE/NAME_HERE, we get {"error":"Permission denied"}

**Content of npmrc file**
`registry=https://registry.npmjs.org/
@ORG_NAME_HERE:registry=https://npm.pkg.github.com/ORG_NAME_HERE
//npm.pkg.github.com/:_authToken=*******
legacy-peer-deps=true
python=C:\Python27\python.exe
`

**Comment by kspearsempire** [comment, marked as answer]
I just discovered yesterday via the GitHub post [Deprecation Notice: GraphQL for Packages](https://github.blog/changelog/2022-08-18-deprecation-notice-graphql-for-packages/) that this functionality has been deprecated in GraphQL and replaced with REST. I have already tested and validated that it is working with REST. It is unfortunate that we discovered this on the 2nd, while this information was not made public until several days later.

**Comment by SGA-masatoshi-ohta** [comment]
We prefer GraphQL over REST...
